code/procs/jobprocs.dm

```diff
@@ -676,39 +676,40 @@ var/global/totally_random_jobs = FALSE
 		qdel(pda)
 
 	var/T = pick(trinket_safelist)
-	var/obj/item/trinket = null
+	var/list/obj/item/trinkets = list()
 
 	if (src.traitHolder && src.traitHolder.hasTrait("pawnstar"))
```

Comment:
can you check that the traitholder exists at the top and remove all these dumb useless checks? really gross

Reply:
[af4cff0](https://github.com/goonstation/goonstation/pull/16352/commits/af4cff0762dd37ec9f11090894ae849e3ed1a1b4)